src/service/extension-location.js

```diff
@@ -28,20 +28,44 @@ import {urls} from '../config';
  */
 let ExtensionInfoDef;
 
+/**
+ * Dynamic AMP components
+ */
+const AMP_DYNAMIC_COMPONENTS = ['amp-geo'];
+
+/**
+ * Determine whether an extension is a dynamic AMP component
+ * @param {string} extensionId
+ * @return {boolean}
+ */
+function isDynamicComponent(extensionId) {
+  return (
+    Boolean(extensionId) && AMP_DYNAMIC_COMPONENTS.indexOf(extensionId) >= 0
```

Comment:
Is case sensitivity important? E.g. `Element.tagName` is always uppercase.

Reply:
I'd argue this is a correct implementation. Fuzzy matching on an ID feels sloppy.

Reply:
Commit cc4f0ed allows fuzzy matching.